### DISCUSSION TITLE
When change docstatus from working to provisional, a time-out error appear

### DISCUSSION THREAD

**Discussion by saraa-costa** [joaomlourenco/novathesis#456] [General]
Hi, 
I´m using overleaf and when I change the documento status from working to provisional or final, a time-out error appear.
![image](https://github.com/joaomlourenco/novathesis/assets/131374649/f967551a-46d7-49ea-85c6-c443c9c0b29f)

**Comment by joaomlourenco** [comment]
Compile again. Usually it works at the second or third attempt.

**Reply by saraa-costa** [reply]
I already try it and doesn’t work :/

**Comment by joaomlourenco** [comment]
Are you using a "new" template, or you added your own chapters with text, tables and/or images?

**Reply by saraa-costa** [reply]
Yes, I add two new chapters but I only can run 4 chapters ,i.e, if I comment chapter 5 and conclusion or comment chapter 4 and 3, overleaf compile without errors, but if I run all chapters (I have 6) overleaf gives that time-out error.

**Comment by joaomlourenco** [comment]
The free version of Overleaf has a (short) timeout.  You cannot compile a longer or more complex document (i.e., with many figures or tables) in the free version.

You have to either pay for a student's Overleaf license or install LaTeX in your own computer and work locally.
